.github/workflows/ci.yml

```
@@ -16,7 +16,7 @@ jobs:
     runs-on: ubuntu-latest
     steps:
     - uses: actions/checkout@v3
-    - uses: actions/setup-python@v2
+    - uses: actions/setup-python@v4
```

```suggestion
    - uses: actions/setup-python@v4
      with:
        python-version: "3.x"
```

Required now, see https://github.com/actions/setup-python/issues/421 :(